collective/ogtags/browser/viewlets.py

```diff
@@ -70,40 +70,36 @@ def meta_tags(self):
         return tags
 
     def image_tags(self):
+        tags = []
+        context = aq_inner(self.context)
         try:
             self.settings = getUtility(
                 IRegistry).forInterface(IOGTagsControlPanel)
         except (ComponentLookupError, KeyError):
             return
         if not self.settings.enabled:
             return
-        tags = []
-        context = aq_inner(self.context)
+        default_image = self.default_image(self.settings.default_img)
+        try:
+            image_provider = IOGTagsImageProvider(context)
+        except TypeError:
+            return default_image
         try:
-            scales = context.restrictedTraverse('@@images', None)
+            image, fieldname = image_provider.getOGTagsImage()
+        except (AttributeError, TypeError):
```

Comment:
I think you may have missed this comment, or you changed a different try/except. No matter, I will change that after merge.